src/scores/probability/crps_impl.py

```diff
@@ -3,11 +3,12 @@
 The primary method, `crps_cdf` is imported into the probability module to be part of the probability API
```

Comment:
Either delete this line or add `crps_for_ensemble`

Reply:
added